spec/requests/api/v1/orders_request_spec.rb

```diff
@@ -0,0 +1,163 @@
+require 'rails_helper'
+
+describe 'Orders API', :type => :request do
+  context 'with a correct API key' do
+    it 'creates an order for a successful request' do
+      distributor = Distributor.create!(name: 'RNDC',
+                                        address: '3319 Arapahoe st, Denver, CO',
+                                        code: 'CODE1234',
+                                        api_key: 'jgn983hy48thw9begh98h4539h4',
+                                        password: 'password'
+                                        )
+
+      item_1 = create(:item, distributor: distributor)
+      item_2 = create(:item, distributor: distributor)
+      business = create(:business, api_key: 'jgn983hy48thw9begh98h4539h4', distributor: distributor)
```

Comment:
You are trying to set the same api_key for the business as the one the distributor has, they have to be different!